frontend/src/components/Header/Header.tsx

```diff
@@ -61,18 +61,12 @@ function AssetPicker() {
             setAllPlantsMap(mapping)
         })
     }, [])
-    let savedAsset = sessionStorage.getItem('assetString')
-    let initialOption = ''
-    if (savedAsset != null) {
-        initialOption = savedAsset
-        switchAsset(savedAsset)
-    }
     const mappedOptions = allPlantsMap ? allPlantsMap : new Map<string, string>()
     return (
         <Autocomplete
             options={Array.from(mappedOptions.keys()).sort()}
             label=""
-            initialSelectedOptions={[initialOption]}
+            initialSelectedOptions={[asset]}
```

Comment:
This would involve a larger change where asset would be renamed to assetCode in assetContext and everywhere assetContext is used. Is this a renaming we want?

Reply:
If asset is a string and not an object, yes I think so